packages/web-forms/src/assets/css/style.scss

```diff
@@ -25,8 +25,11 @@
 	--odk-primary-lighter-background-color: var(--p-primary-50);
 	--odk-primary-border-color: var(--p-primary-500);
 
-	--odk-error-text-color: var(--p-message-error-color);
-	--odk-error-background-color: var(--p-message-error-background);
+	--odk-error-text-color: var(--p-red-600);
```

Comment:
I realized that `--p-message-error-background` is not always available if the message component is unused. Better use the base color variable from PrimeVue